contents.json

```diff
@@ -10488,6 +10488,24 @@
       "screenshots": [
         "http://a3.mzstatic.com/us/r30/Purple3/v4/be/ea/39/beea3978-efb6-07ef-aefb-3d01e09531d9/screen696x696.jpeg"
       ]
+    },
+        {
+      "title": "Blahker",
+      "category-ids": [
+        "content-blocking"
+      ],
+      "description": "Safari content blocker for interstitial ads, mainly for websites in Taiwan",
+      "license": "mit",
+      "source": "https://github.com/ethanhuang13/blahker",
+      "itunes": "https://itunes.apple.com/tw/app/blahker-ba-la-ke-gai-ban-guang/id1182699267",
+      "date_added": "Apr 2 2018 +0800",
+      "suggested_by": "@ethanhuang13",
+      "stars": 4.9,
```

Comment:
`stars` would be the number of stars on GitHub, so 70 at the moment

Reply:
Oh, OK~